source/includes/trip-api.md

```diff
@@ -790,7 +799,7 @@ Status code | Description
 
 ## Cancel endpoint
 
-This endpoint is used to cancel a booked trip. Only trips that have departure more than 24 hours in the future can be cancelled - this threshold is a subject to change.
+This endpoint is used to cancel a booked trip. For private trips only trips that have departure more than 24 hours in the future can be cancelled - this threshold is a subject to change. The shared trips can be cancelled at any time before departure, and compensated according to the selected [cancellation policy](#cancellationpolicytype).
```

Comment:
This is problematic, current endpoint does not return information to the partner how much he needs to pay for the cancellation. Should be extended.

Reply:
Proposing to return `penalty` and `currency` (ae06991), which we already calculated in the code.

Reply:
Looks good, will need to also update code for private trip logic to always return this. This API spec MR should be merged when the whole pool epic + this cancel change goes live.